manim/__main__.py

```diff
@@ -15,7 +15,7 @@ def exit_early(ctx, param, value):
         sys.exit()
 
 
-console.print(f"Manim Community [green]v{__version__}[/green]")
+console.print(f"Manim Community [green]v{__version__}[/green]\n")
```

Comment:
Why wasn't the newline necessary before?

Reply:
It was more of a personal preference, having a space between the title and the descriptions.

Reply:
I'm okay with that